src/Hl7.Fhir.Support.Poco/Introspection/PropertyMapping.cs

```diff
@@ -279,21 +291,9 @@ ITypeSerializationInfo[] IElementDefinitionSummary.Type
 
         int IElementDefinitionSummary.Order => Order;
 
-        /// <summary>
-        /// Given an instance of the parent class, gets the value for this property.
-        /// </summary>
-        public object GetValue(object instance) => Getter(instance);
```

Comment:
Interestingly, you will not get any compile errors over this, see, there is no syntactic difference between calling a function that calls a "sub" function and returs its return value or returning that "sub" function and immediately calling it ;-)   But it was not really intentional, I'll change it.

Reply:
It is certainly a different interface, but it will not cause a compile-time error, and it does exactly the same thing.  But I did not want to change it, so I'll see if I can revert it.